portfolio/src/main/webapp/index.html

```diff
@@ -2,17 +2,76 @@
 <html>
   <head>
     <meta charset="UTF-8">
-    <title>My Portfolio</title>
+    <title>Kristen's Portfolio</title>
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
   </head>
   <body>
     <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
+      <h1>Kristen's Portfolio</h1>
+      <div class="tab">
+        <button class="tablinks active" onclick="openTab(event,'home')">Home</button>
+        <button class="tablinks" onclick="openTab(event,'about-me')">About Me</button>
+        <button class="tablinks" onclick="openTab(event,'my-projects')">My Projects</button>
+        <button class="tablinks" onclick="openTab(event,'fun-facts')">Fun Facts</button>
+        <button class="tablinks" onclick="openTab(event,'travel-blog')">Travel Blog</button>
+        <button class="tablinks" onclick="openTab(event,'contact-me')">Contact Me</button>
+      </div>
+        <div id="home" class="tabcontent">
+        <h3> Hi there, and welcome to my portfolio! </h3>
+        </div>
+        <div id="about-me" class="tabcontent">
+            <h2> About Me:</h2>
```

Comment:
Remove the space between the tag and the start of the text. If you wanted that effect to stay, use css positioning

Reply:
Did this on accident, resolved in latest commit